README.md

```diff
@@ -46,46 +20,28 @@ ServicePulse uses [npm](https://www.npmjs.com/) and [Bower](https://bower.io/) a
 
 #### Install dependencies
 
-Install the following dependencies if you don't have them installed yet
+Install the following dependencies if you don't have them installed yet.
 
 - [Git for Windows](https://gitforwindows.org/)
 - [Node.js](https://nodejs.org/en/download/)
-  - **NOTE:** It is good practice when installing or updating node.js on your local environment to also check and make sure that the node.js version for the `ci.yml` and `release.yml` workflows match the version of your local environment. Look for the step `name: Set up Node.js`
+  - **NOTE:** It is good practice when installing or updating node.js on your local environment also to check and make sure that the node.js version for the `ci.yml` and `release.yml` workflows match the version of your local environment. Look for the step `name: Set up Node.js`
 - [Docker](https://docs.docker.com/get-docker/)
-  - **NOTE:** If running docker for windows, the current user needs to be in the docker-users local group. If your windows account is different to the installation (admin) account, then you will need to manually add your user to the group. See https://docs.docker.com/desktop/faqs/windowsfaqs/#why-do-i-see-the-docker-desktop-access-denied-error-message-when-i-try-to-start-docker-desktop
+  - **NOTE:** If running docker for Windows, the current user needs to be in the docker-users local group. If your Windows account differs from the installation (admin) account, you must manually add your user to the group. See <https://docs.docker.com/desktop/faqs/windowsfaqs/#why-do-i-see-the-docker-desktop-access-denied-error-message-when-i-try-to-start-docker-desktop>
```

Comment:
```suggestion

```

Reply:
Since we no longer require to start the docker instance as a proxy, we can also remove docker as a dependency for development